content/en/observability_pipelines/legacy/_index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: (LEGACY) Observability Pipelines Documentation
+kind: Documentation
+aliases:
+  - /integrations/observability_pipelines/
+further_reading:
+  - link: /observability_pipelines/legacy/setup/
+    tag: Documentation
+    text: Set up Observability Pipelines
+  - link: https://www.datadoghq.com/blog/datadog-observability-pipelines/
+    tag: Blog
+    text: Take control of your telemetry data with Observability Pipelines
+  - link: /observability_pipelines/legacy/configurations/
+    tag: Documentation
+    text: Learn more about Observability Pipelines configurations
+cascade:
+    algolia:
+        rank: 70
+private: true
+cascade:
+    private: true
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Observability Pipelines is not available on the US1-FED Datadog site.</div>
+{{< /site-region >}}
+
+<div class="alert alert-danger">If you upgrade your Observability Pipelines Workers 1.8 or earlier to 2.0 or later, your existing pipelines will break. Do not upgrade your Workers if you want to continue using the legacy Observability Pipelines. If you want to use Worker 2.0 or later, you must migrate your Worker 1.8 or earlier pipelines to Worker 2.x.</div>
```

Comment:
I'm guessing that `migrating` and `upgrading` are two different processes. Is there more information on how to migrate to the new worker vs. upgrading?